listener/listener.go

```diff
@@ -267,6 +268,30 @@ func (l *Listener) Close(ctx context.Context) error {
 	return nil
 }
 
+// GetActiveMessageCount gets the active message count of a topic subscription
```

Comment:
Add a warning that this is at least 10x expsensive as a recieve call.

Reply:
Added a warning.